tasmota/language/en_GB.h

```diff
@@ -1315,4 +1319,16 @@
 #define D_CHARGING                        "Charging"
 #define D_CAPACITY                        "Capacity"
 
+// xdrv_93_esp32_gnss_ntrip.ino
```

Comment:
You need to copy this section in all languages files (use search/replace multi-line) to make it at once.